pkg/feature/feature.go

```diff
@@ -253,3 +275,32 @@ func (f *Feature) createResourceTracker() error {
 
 	return nil
 }
+
+func (f *Feature) UpdateFeatureTrackerStatus(condType conditionsv1.ConditionType, status corev1.ConditionStatus, reason featurev1.FeaturePhase, message string) {
```

Comment:
I think we need to rework this one a little bit. It's not `nil` safe. Let's wait for #812 and do it together.

Reply:
[bc7d71e](https://github.com/opendatahub-io/opendatahub-operator/pull/801/commits/bc7d71e3c202b3a69f5ca72e60b637411418ea29)